sdk/schema/fieldname/names.go

```diff
@@ -17,6 +17,7 @@ const (
 	AppSecret       = sdk.FieldName("App Secret")
 	AppToken        = sdk.FieldName("App Token")
 	AuthToken       = sdk.FieldName("Auth Token")
+	Authtoken       = sdk.FieldName("Authtoken")
```

Comment:
We should also add it to the `ListAll` function below.

Reply:
Done, thanks for catching that